src/Wrapper/AssetPairWrapper.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using KrakenCore.Models;
+using KrakenCore.Utils;
+
+namespace KrakenCore.Wrapper
+{
+    public class AssetPairWrapper : WrapperBase
+    {
+        public string Key { get; }
+
+        /// <summary>
+        /// Work in progress
+        /// </summary>
+        public AssetPair Model { get; internal set; }
+
+        public TickerInfo Ticker { get; internal set; }
+
+        internal AssetPairWrapper(KrakenWrapper client, string key, AssetPair model = null) : base(client)
+        {
+            this.Key = key;
```

Comment:
From a compiler perspective, yes.
As a reader "this." clarifies what is going on by making a distinction between the argument and instance variable. 
Otherwise you have to look careful to see the difference.

    Key = key;
    key = Key;

What do you think?

Reply:
It's a matter of code style consistency across the solution. I don't mind either way, it's just that the existing code has avoided the usage of `this`. If it was used for readability purposes, it should be applied everywhere.